{{cookiecutter.project_shortname}}/Dockerfile.base

```diff
@@ -2,4 +2,4 @@
 FROM inveniosoftware/centos7-python:3.6
 
 COPY Pipfile Pipfile.lock ./
-RUN pipenv install --deploy
+RUN pipenv install --deploy --system
```

Comment:
What's the effect of ``--system``?

Reply:
It installs dependencies outside virtualenv.